src/main/java/com/denizenscript/clientizen/objects/MaterialTag.java

```diff
@@ -0,0 +1,138 @@
+package com.denizenscript.clientizen.objects;
+
+import com.denizenscript.clientizen.util.Utilities;
+import com.denizenscript.denizencore.objects.*;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.properties.PropertyParser;
+import com.denizenscript.denizencore.tags.Attribute;
+import com.denizenscript.denizencore.tags.ObjectTagProcessor;
+import com.denizenscript.denizencore.tags.TagContext;
+import com.denizenscript.denizencore.utilities.CoreUtilities;
+import com.denizenscript.denizencore.utilities.debugging.Debug;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.item.Item;
+import net.minecraft.registry.Registries;
+import net.minecraft.state.State;
+import net.minecraft.util.Identifier;
+
+public class MaterialTag implements ObjectTag, Adjustable {
+
+    public BlockState state;
+    public Item item;
```

Comment:
There should be a comment here explaining why both of these fields are present.

Reply:
Added